test/tests/parallel/set1/spam_virus/rspamd.bats

```diff
@@ -0,0 +1,98 @@
+load "${REPOSITORY_ROOT}/test/helper/setup"
+load "${REPOSITORY_ROOT}/test/helper/common"
+
+BATS_TEST_NAME_PREFIX='[rspamd] '
+CONTAINER_NAME='dms-test_rspamd'
+
+function setup_file() {
+  _init_with_defaults
+
+  # Comment for maintainers about `PERMIT_DOCKER=host`:
+  # https://github.com/docker-mailserver/docker-mailserver/pull/2815/files#r991087509
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_CLAMAV=1
+    --env ENABLE_RSPAMD=1
+    --env CLAMAV_MESSAGE_SIZE_LIMIT=30M
```

Comment:
Is this ENV relevant to the test?

Reply:
Not at all :D

Reply:
Done in [d604127](https://github.com/docker-mailserver/docker-mailserver/pull/3016/commits/d604127f5ebf09ccf97281c104f0d552bbcfd52b)